custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -781,23 +779,40 @@ def check_prediction_side_effects(self):
 
             self._schema_validator.validate_outputs(preds_sample)
 
-            matches = np.isclose(preds_full_subset, preds_sample, rtol=rtol, atol=atol)
-            if not np.all(matches):
-                if is_sparse:
-                    _, __tempfile_sample = mkstemp(suffix=InputFormatExtension.MTX)
-                    sparse_mat = vstack(x[0] for x in data_subset.values)
```

Comment:
Looks like the fix was by removing this `vstack`and just using `data_subset.sparse` directly?

Reply:
That is correct.  That statement gets the first value instead of the row, which can't be stacked.  It doesn't make sense to do it this way, the built in method does what we want.